internal/topo/planner/planner_test.go

```diff
@@ -1870,6 +1898,444 @@ func Test_createLogicalPlan(t *testing.T) {
 				},
 			}.Init(),
 		},
+		{ // 19
+			sql: `SELECT * EXCEPT(id1, name), meta(device) FROM src1`,
```

Comment:
What about `select name, * EXCEPT(id1, name)`

Reply:
For `select name, * EXCEPT(id1, name)`, streamFields will contain "temp", "myarray", and "name"; similarly, for` SELECT id1 FROM src1 GROUP BY TUMBLINGWINDOW(ss, 10) HAVING count(* EXCEPT(id1, name)) > 0`, streamFields will contain "temp", "myarray", and "id1".